src/main/java/Utilities/CommandParser.java

```diff
@@ -0,0 +1,96 @@
+package utilities;
+
+import exceptions.BadDescriptionException;
+import exceptions.DateTimeException;
+import exceptions.UnknownCommandException;
+
+import java.time.format.DateTimeParseException;
+
+import tasks.Deadlines;
+import tasks.Event;
+import tasks.Task;
+import tasks.TaskTypes;
+import tasks.ToDos;
+
+public class CommandParser {
+    /**
+     * Parses and performs actions for the given command.
+     *
+     * @param input String command for processing.
+     * @param tl TaskList class for maintaining tasks.
+     * @param store Storage class for writing to files.
+     * @return String value for user response.
```

Comment:
Good use of javadoc annotations to describe what each parameter is and the return value of the method!